evolver/app/tests/test_calibration.py

```diff
@@ -15,12 +15,12 @@ def test_get_calibration_status(self):
             hardware={
                 "temp": NoOpSensorDriver(
                     name="temp",
-                    calibrator=NoOpCalibrator(state={"status": "calibrated"}),  # Mock calibrator state
```

Comment:
Removed the `state` attribute on the Calibrator, it's now in the` Calibrator.CalibrationData`